src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
```

Comment:
I guess that better is delete similar comments in the next tasks. Everything else is fine. Good job!